modules/custom/az_paragraphs/az_paragraphs.install

```diff
@@ -28,6 +28,10 @@ function az_paragraphs_update_920301(&$sandbox) {
     ->range($sandbox['current'], $sandbox['current'] + $paragraphs_per_batch)
     ->execute();
 
+  if (empty($paragraph_ids)) {
```

Comment:
Maybe we should move this inside the if block that initializes the sandbox (with the count query)?

Reply:
(I'm just thinking having it here could cause the message to get logged on sites that do have paragraphs)

Reply:
Checking